src/Employer/Employer.Web/Views/ManageNotifications/ManageNotifications.cshtml

```diff
@@ -0,0 +1,107 @@
+                            </label>
+                        </div>
+                        <div class="govuk-checkboxes__item">
+                            <input class="govuk-checkboxes__input" asp-for="IsVacancyClosingSoonSelected" id="notifications-closing" type="checkbox">
+                            <label class="govuk-label govuk-checkboxes__label" for="notifications-closing">
+                                <span class="govuk-visually-hidden">Get notifications when </span>Vacancies are closing soon
+                            </label>
+                        </div>
+                        <div class="govuk-checkboxes__item govuk-checkboxes--conditional">
+                            <input class="govuk-checkboxes__input" asp-for="IsApplicationSubmittedSelected" id="notifications-new" type="checkbox" data-aria-controls="conditional-when-conditional">
+                            <label class="govuk-label govuk-checkboxes__label" for="notifications-new">
+                                <span class="govuk-visually-hidden">Get notifications when </span>There are new applications
+                            </label>
+                        </div>
+                        <div class="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden" id="conditional-when-conditional">
+                            <fieldset class="govuk-fieldset" aria-describedby="when-conditional-hint">
+                                <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+                                    <h1 class="govuk-fieldset__heading">
+                                        How often do you want to be sent new application emails?
+                                    </h1>
+                                </legend>
+                                <span esfa-validation-message-for="NotificationFrequency" class="govuk-error-message"></span>
+                                <div class="govuk-radios" data-module="radios">
+                                    <div class="govuk-radios__item">
+                                        <input class="govuk-radios__input" value="Immediately" asp-for="NotificationFrequency" id="notification-frequency-immediately" type="radio">
+                                        <label class="govuk-label govuk-radios__label" for="notification-frequency-immediately">
+                                            Immediately
+                                        </label>
+                                    </div>
+                                    <div class="govuk-radios__item">
+                                        <input class="govuk-radios__input" value="Daily" asp-for="NotificationFrequency" id="notification-frequency-daily" type="radio" >
+                                        <label class="govuk-label govuk-radios__label" for="notification-frequency-daily">
+                                            Daily
```

Comment:
Whoever had the idea of calling these options single one words is a genius

Reply:
Could you use the enum value to output the display text here is what I was hinting at and the genius is me.